modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/dnd_table/Cell.js

```diff
@@ -97,6 +105,7 @@ function Cell({children, className, columnName, expand, heading, resizable}) {
 			className={classNames(
 				heading ? 'dnd-th' : 'dnd-td',
 				expand && 'expand',
+				isLastCell && 'hide-on-hover',
```

Comment:
I don't think we need `isLastCell`, or `hide-on-hover`. We should be able to set this just with something like `.with-quick-actions tr:last-child`.

Also, hiding the last column is the simplest solution, but it may not be enough. In the [Lexicon samples](https://liferay.design/lexicon/core-components/table/), two last rows are hidden. Maybe we need some JS logic that hides based on column widths?

Reply:
pushed!